test/tools/unified-spec-runner/unified-utils.ts

```diff
@@ -1,30 +1,31 @@
+/* eslint-disable @typescript-eslint/no-restricted-imports */
 import { EJSON } from 'bson';
 import { expect } from 'chai';
 import ConnectionString from 'mongodb-connection-string-url';
 import { gte as semverGte, lte as semverLte } from 'semver';
 import { isDeepStrictEqual } from 'util';
 
+import { ClientEncryption } from '../../../src/client-side-encryption/clientEncryption';
```

Comment:
Duplicate identifier error here, how's this working?

Reply:
Our tests aren't compiled